tests/api/counties_test.py

```diff
@@ -5,61 +5,69 @@ class TestApiCounties:
     def test_get_county(self, client):
         """Start with a blank database."""
 
-        data = client.get('/api/counties/1').get_json()
+        data = client.get("/api/counties/1").get_json()
```

Comment:
Similar to the import question, what's our repo's style choice on single versus double quotes? I prefer double quotes everywhere for strings so they dont conflict with apostrophes.. but I notice a lot of people use single quotes. We should have a basic style guide for our repo lol

Reply:
Double quotes are _so_ much easier ... because then you can type *can't* without changing your quote style.

I think I was just lazy here the first time (I copied and pasted the code for this test).